packages/sitecore-jss/package.json

```diff
@@ -47,15 +47,14 @@
     "eslint": "^8.56.0",
     "eslint-plugin-jsdoc": "48.7.0",
     "mocha": "^10.2.0",
-    "nock": "^13.0.5",
+    "nock": "14.0.0-beta.7",
```

Comment:
Just to comment on why this specific version is used, for history.
Using beta version to fix this issue: https://github.com/nock/nock/issues/2397 and allow testing for component library service
Using specific beta.7 version to not encounter this error: https://github.com/nock/nock/issues/2789